redhat-developer/developers.redhat.com
# [SURGE 3037] Video Hero

### JIRA Issue Link
* https://github.com/redhat-developer/developers.redhat.com/issues/3037

### Verification Process
1. View Video Hero assembly in a node context with one video
2. Verify it matches the FE mocks (https://redhat-developer.github.io/rhd-frontend/components/video-hero/) with:
- One video
- Two videos
- Background image
- No background image
